libcodechecker/arg_handler.py

```
@@ -134,7 +134,7 @@ def handle_server(args):
                              str(instance['port'])))
 
         print("Your running CodeChecker servers:")
-        util.print_table(rows)
+        print(util.twodim_to_table(rows))
```

Possible `TypeError` exceptions are not handled.